gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -328,6 +330,7 @@ func (vm *VMKeeper) AddPackage(ctx sdk.Context, msg MsgAddPackage) (err error) {
 	creator := msg.Creator
 	pkgPath := msg.Package.Path
 	memPkg := msg.Package
+	send := msg.Send
```

Comment:
note to self: is send amount segregated from storage deposits?
(the realm should not be able to spend its storage deposits)

Reply:
good catch, it was not and is fixed